user/serializers.py

```diff
@@ -22,3 +23,41 @@ def update(self, instance, validated_data):
             user.save()
 
         return user
+
+
+class AuthTokenSerializer(serializers.Serializer):
+        email = serializers.CharField(
+            label=("email",),
+            write_only=True
+        )
+        password = serializers.CharField(
+            label=("Password",),
+            style={"input_type": "password"},
+            trim_whitespace=False,
+            write_only=True
+        )
+        token = serializers.CharField(
+            label=("Token",),
+            read_only=True
+        )
+
+        def validate(self, attrs):
+            username = attrs.get("email")
+            password = attrs.get("password")
+
+            if username and password:
+                user = authenticate(request=self.context.get("request"),
+                                    username=username, password=password)
+
+                # The authenticate call simply returns None for is_active=False
+                # users. (Assuming the default ModelBackend authentication
+                # backend.)
+                if not user:
+                    msg = "Unable to log in with provided credentials."
+                    raise serializers.ValidationError(msg, code="authorization")
+            else:
+                msg = "Must include 'username' and 'password'."
```

Comment:
The error message should refer to 'email' instead of 'username' to accurately reflect the field being used for authentication. Change the message to "Must include 'email' and 'password'."